CONTRIBUTING.md

```diff
@@ -4,13 +4,33 @@ This is a [Jazzband](https://jazzband.co/) project. By contributing you agree
 to abide by the [Contributor Code of Conduct](https://jazzband.co/about/conduct)
 and follow the [guidelines](https://jazzband.co/about/guidelines).
 
+## Getting started
+
+Install tox:
+
+```console
+$ python -m pip install tox --user
+```
+
+Create a virtual environment, with `pip-tools` in development mode and its test dependencies,
+using your current python version:
+
+```console
+$ tox --devenv .venv -e py
+```
+
+Activate the virtual environment:
+
+```console
+$ source .venv/bin/activate
+```
+
 ## Project Contribution Guidelines
 
 Here are a few additional or emphasized guidelines to follow when contributing to pip-tools:
 
-- Install pip-tools in development mode and its test dependencies with `pip install -e .[testing]`.
 - Check with `tox -e checkqa` to see your changes are not breaking the style conventions.
```

Comment:
We could drop this line since `pre-commit.ci` checks and autofixes QA issues. What do you think?

Reply:
Yes that's a nice catch.
Also if we suggest to run `tox -p all` then I think it will still be run?

Reply:
I think it's a good idea to have `-p` there. The ultimate goal is to let people know how to run the same test matrix that the CI runs locally. And running linters locally can also be useful because people often want to reproduce the offenses in their env to be able to address them. Rerunning the whole CI on every typo fix push is wasteful.